inginious-problems-blockly/static/studio_blockly.js

```diff
@@ -14,13 +14,83 @@ function studio_init_template_blockly(well, pid, problem) {
     var workspaceEditor = registerCodeEditor($('#workspace-' + pid)[0], 'xml', 10);
     workspaceEditor.setValue("workspace" in problem ? problem.workspace : "<xml></xml>");
 
-    var options = "options" in problem ? problem.options : {};
+    var options = "options" in problem ? problem.options : {
+        "collapse" : false,
+        "comments" : false,
+        "disable" : false,
+        "maxBlocks" : "Infinity",
+        "trashcan" : false,
+        "horizontalLayout" : false,
+        "toolboxPosition" : "start",
+        "css" : true,
+        "rtl" : false,
+        "scrollbars" : true,
+        "media" : "plugins/blockly/static/blockly/media/",
```

Comment:
It would be great if this was automatically managed by the plugin to ease the task edition. I'd like to see this in a new PR.